### DISCUSSION TITLE
Published scene falling over / falls to the floor, how to fix?

### DISCUSSION THREAD

**Discussion by LovesPictures** [mozilla/hubs#2803] [General Questions & Discussion] [answered]
Hi
When published the the scene appears to fall over/fall to the floor. 
What's going on here and how do I fix it please?

Thanks

**Comment by emclaren** [comment, marked as answer]
Hi there! I imagine this is frustrating. I'd love to offer whatever guidance I can, but I'm not totally sure I understand what is happening. Are you able to provide any additional information, or drag and drop a screencapture into a reply to demonstrate what you mean?

**Reply by LovesPictures** [reply]
Hi here's a short .mp4 video of the problem.

[fallingOver.zip](https://github.com/mozilla/hubs/files/5034702/fallingOver.zip)

**Reply by emclaren** [reply]
Thanks for the video, I see what you mean that it is falling over. 

I've not seen anything like this before, so I'm not sure why it's happening. Would you be able to publish this scene with the "remixable" check box ticked and share it with me here (or if you don't want it posted publicly at hubs@mozilla.com  ). I would be happy to take a look and see if I can figure out why this is happening.

**Reply by LovesPictures** [reply]
Hello, Thanks,

When I remix will a version of this always available? as I don't want to have it be re-mixable just yet...

**Reply by emclaren** [reply]
you can set it to be non-remixable when you upload it next.  

Alternatively, if you dont want to make it remixable, feel free to go to the dropdown, find then "export legacy .spoke file" option and email it to us at hubs@mozilla.com

**Reply by LovesPictures** [reply]
Hi, Yes, 

I'm sending over now using the "export legacy .spoke file"

Thank you

**Reply by emclaren** [reply]
Thanks for sending this over. We took at look at your scene and it seems like this is a bug. There is an easy fix for your scene though. 

Its being caused by the spawn point, the scale has been changed from the default. If you set it to 1,1,1 it should work again. 

We've tracked the bug in the following issue: https://github.com/mozilla/hubs/issues/2822
